homeassistant/components/backup/manager.py

```diff
@@ -460,7 +627,7 @@ async def async_create_backup(
             on_progress=self.async_on_backup_event,
             password=password,
         )
-        self.finish_backup_task = self.hass.async_create_task(
+        self.backup_post_task = self.hass.async_create_task(
```

Comment:
This name is maybe too similar to `async_post_backup_actions` so that it could be confused. Maybe keep `finish` or some other variant of that meaning.

Reply:
```suggestion
        self.backup_finish_task = self.hass.async_create_task(
```